### ISSUE TITLE
Classify command doesn't work when directly passed file as input

### ISSUE THREAD

**Issue by ernfrid** [hall-lab/svtools#145]
I see errors like:

```
Traceback (most recent call last):
  File "/tmp/dlarson_testing/scripts/sv_classifier.py", line 599, in <module>
    sys.exit(args.entry_point(args))
  File "/tmp/dlarson_testing/scripts/sv_classifier.py", line 593, in run_from_args
    with su.InputStream(args.vcf_in) as stream:
  File "/tmp/dlarson_testing/miniconda/envs/svtools0.2.0/lib/python2.7/site-packages/svtools-0.2.0-py2.7.egg/svtools/utils.py", line 10, in __init__
AttributeError: 'file' object has no attribute 'endswith'
```

**Comment by abelhj** [comment]
Sorry.  Fixed this but also need to get rid of a pandas warning.

**Comment by ernfrid** [comment]
This was fixed in #148.
